pytorch_lightning/trainer/deprecated_api.py

```diff
@@ -24,7 +24,7 @@ class DeprecatedDistDeviceAttributes:
     @property
     def on_cpu(self) -> bool:
         # rank_zero_warn("Internal: `on_cpu` is deprecated in v1.1 and will be removed in v1.2.", DeprecationWarning)
```

Comment:
not related to PR, but should't the deprecation and removal here be 0.2 apart??

Reply:
yes, it is in #5297 #5300